changelog/6870.feature.rst

```diff
@@ -0,0 +1 @@
+New ``Config.invocation_args`` attribute containing the unchanged arguments passed to ``pytest.main()``.
```

Comment:
Let's add a remark here:

```suggestion
New ``Config.invocation_args`` attribute containing the unchanged arguments passed to ``pytest.main()``.

Remark: while this is technically a new feature and according to our `policy <https://docs.pytest.org/en/latest/py27-py34-deprecation.html#what-goes-into-4-6-x-releases>`_ it should not have been backported, we have opened an exception in this particular case because it fixes a serious interaction with ``pytest-xdist``, so it can also be considered a bugfix.
```

@asottile what do you think?

Reply:
👍 makes it much more clear that this is one-off and not a greenlight to resume features on 4.6

Reply:
Remark added, it makes a lot of sense.

Reply:
Oh just noticed this was not added in the end.

No worries, I will update directly on #7199. 👍

Reply:
I committed it locally and never pushed 🤦. Sorry about that

Reply:
No problem 😁 

4.6.10 has been released btw! 👍